src/lib.rs

```diff
@@ -1050,35 +1165,29 @@ mod universe_tests {
     #[test]
     fn new_universe_with_bad_dims() {
 
-        let player0_writable = Region::new(100, 70, 34, 16);   // used for the glider gun and predefined patterns
-        let player1_writable = Region::new(0, 0, 80, 80);
-        let writable_regions = vec![player0_writable, player1_writable];
-
-        let uni_result1 = Universe::new(255,   // width
-                                        128,   // height
-                                        true,  // server_mode
-                                        16,    // history
-                                        2,     // players
-                                        writable_regions.clone()
-                                      );
+        let player0 = PlayerBuilder::new(Region::new(100, 70, 34, 16));   // used for the glider gun and predefined patterns
```

Comment:
It might be cleaner to do `PlayerBuilder::new().region(100, 70, 34, 16)`, since we'll probably have other player creation options in the future...

Reply:
Mm yeah I agree with ya.